src/nodes.coffee

```diff
@@ -542,9 +556,10 @@ exports.Block = class Block extends Base
       break
     this
 
-  # A **Block** is the only node that can serve as the root.
-  compileToFragments: (o = {}, level) ->
-    if o.scope then super o, level else @compileRoot o
+  compile: (o, lvl) ->
+    return new Root(this).withLocationDataFrom(this).compile o, lvl unless o.scope
```

Comment:
As described, `Block::compile()` provides a backwards-compatibility "shim" to wrap itself in a `new Root` if need be